topostats/theme.py

```diff
@@ -10,17 +10,18 @@
 
 
 class Colormap:
-    """Class for setting the COlormap"""
+    """Class for setting the COlormap."""
```

Comment:
Typo: "Colormap" rather than "COlormap"

Reply:
Ta. Personally I'd also rather see the Anglicised spelling too, but am not going to make an issue of it particularly as Matplotlib uses `color` rather than `colour`.

Reply:
I thought the same, perhaps best to stick with Matplotlib's spelling!